deploy/Offline-version.json

```diff
@@ -0,0 +1 @@
+{"version": "0.2.0"}
```

Comment:
The purpose of this field might be more obvious if we kept it named `min-version`.

(and updated usages accordingly)